_posts/2019-11-19-s2o-how-it-was.markdown

```diff
@@ -0,0 +1,59 @@
+
+* I've never successfully presented it without technical issues; and
+* It typically took 45 minutes to delivery (I only had 25); and
+* It was right before afternoon coffees when everyone is sleepy; and
+* It was usually delivered in an interactive manner which wouldn't translate to the theatre.
+
+Oh, and that it was likely to be recorded for posterity!
+
+Still, I'm never one to shy away from a challenge, so I knuckled down and got to work.
+
+## A little background
+
+![Pete thinking face with banner]({{ site.url }}/img/inception-pipelines/s2o-how-it-was/banner-thinking.jpg)
+
+Let me start with some background. I have previously presented versions of it to several meetups across Australia (as well as internally to a few select groups). Each time I've presented it, something had gone wrong; from using the wrong AWS account, conflicting AWS resources that hadn't been properly removed from past presentations to just flat out technical issues connecting to AV equipment. So, task #1 was making sure I had a run sheet to make sure those possibilities were taken care of.
+
+Next step was to sit down and just write the story of what I wanted to tell. I strongly believe that every time you get on stage, you're there to tell a story. People love stories and quite often remember the storytelling rather than the content. Plus, not many people like long, droning lectures. The act of just sitting down and writing the story meant I could focus on setting the stage, building up towards the climax and finally wrapping it all up in the conclusion.
+
+My talk is a live-code presentation where I take the audience on a journey through what is possible on AWS for Serverless applications. It is a breadth-not-depth, infrastructure-as-code presentation where I start from nothing, deploy an application (including all CI/CD infrastructure) into the cloud and then promptly turn around and try to burn it to the ground.
+
+After getting my story straight, I spent many hours talking to myself reading out the story and refining it. I can only image what people around me thought!! Still, it was invaluable as from here I was able to distil it all down to a bunch of index cards with the points I wanted to cover. It was also invaluable in helping me in timing parts of the talk. Remember, it is a live-coding presentation so knowing where I need to add 'filler' and where to wrap things up was key. I couldn't face the horror of standing in front of 300 people, crickets chirping, having nothing to say while I waited for the code to catch-up. A bonus was that continually presenting it helped immensely in ironing out all the sharp bits that kept poking me; I have confidence the code would deploy and run smoothly without errors and in the allocated time.
```

Comment:
distill

Reply:
From a quick search; a single 'l' is the English spelling; double 'l' is the American. Leaving as is.